### ISSUE TITLE
lb-only: Document standalone L4LB

### ISSUE THREAD

**Issue by brb** [cilium/cilium#17485]

**Comment by oblazek** [comment]
Hi, any WIP? Or it's free to take? 🙂

**Comment by brb** [comment]
cc @qmonnet who is currently working on it.

**Comment by qmonnet** [comment]
Yes I am, I've got something but didn't have time to finish it before the LPC conference. I'll finish and post next week, I can CC you for review/comments when this happens.
